### DISCUSSION TITLE
Explanation of this test?

### DISCUSSION THREAD

**Discussion by benkoshy** [ddnexus/pagy#742] [Q&A] [answered]
Please refer to the test noted here:

https://github.com/ddnexus/pagy/blob/b226ae7b7536281b0c1fb4cada29bbca6db13e93/test/pagy/i18n_locales_test.rb#L30

Here is a reproduction:

```rb
  it 'includes a comment with the pluralization rule and the i18n.rb reference' do
      _(rules).must_include rule, message
      # what does the message variable do?
      # I had added the following random letters and the tests pass perfectly?
      _(rules).must_include rule, "asdf asdf " # passes
    end
```

It seems like it should not pass?

References:

https://ruby-doc.org/stdlib-3.0.2/libdoc/minitest/rdoc/Minitest/Expectations.html#method-i-must_include

**Comment by ddnexus** [comment, marked as answer]
IIRC the `message` is an optional/custom param that gets printed when the test fails. Changing it doesn't affect the outcome of the test, only its caption/description.

In this case it provides custom feedback pointing to the file that failed.
